projects/rocprofiler-systems/source/lib/core/trace_cache/rocpd_post_processing.cpp

```diff
@@ -764,7 +765,7 @@ rocpd_post_processing::post_process_metadata()
             rocpd_agent->logical_node_id, rocpd_agent->logical_node_type_id,
             rocpd_agent->device_id, rocpd_agent->name.c_str(),
             rocpd_agent->model_name.c_str(), rocpd_agent->vendor_name.c_str(),
-            rocpd_agent->product_name.c_str(), "");
+            rocpd_agent->product_name.c_str(), "", rocpd_agent->agent_info.c_str());
```

Comment:
I know it wasn't part of your change, but since we are modifying this file already, can we define a constant for this empty string?
```suggestion
            rocpd_agent->product_name.c_str(), USER_NAME, rocpd_agent->agent_info.c_str());
```

Reply:
To be completely honest, I am not sure what is the purpose of the user name, as far as I see it, the `rocprofv3` is using product name for that field, so we might as well do the same in order to be aligned